xen/xenops_interface.ml

```diff
@@ -433,6 +433,8 @@ module Vif = struct
 
 	type id = string * string
 
+	type static_ip_setting = (string * string) list
```

Comment:
That would be:
`Static ["192.168.1.100/24"; "192.168.1.200/24"] (Some "192.168.1.1")`.